site.yml

```diff
@@ -91,6 +91,9 @@
       copy: src=jobs dest=/var/lib/jenkins/jobs/default-dsl-job/workspace directory_mode=0750 mode=0640 owner=jenkins group=jenkins
     - name: generate jenkins job from dsl seed job
       shell: "curl -X POST 'http://127.0.0.1:8080/job/default-dsl-job/build' -k $(cat ~/jenkins_auth)"
+    - name: wait for creation and disable terraform jenkins job on vagrant boxes
+      shell: "sleep 15 && curl -X POST 'http://127.0.0.1:8080/job/terraform-deploy/disable' -k $(cat ~/jenkins_auth)"
```

Comment:
This doesn't feel right. Do the Ansible jobs also need disabling so that they don't blat the CI environment with unmerged code?

Would an alternative solution be for Vagrant to use a different environment name than "ci"? Does it need to be unique for each person using this repo?